test/parallel/test-require-resolve-opts-paths-relative.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+
+if (!common.isMainThread)
+  common.skip('process.chdir is not available in Workers');
+
+const subdir = fixtures.path('module-require', 'relative', 'subdir');
+
+process.chdir(subdir);
+
+// Parent directory paths (`..`) work as intended
+{
+  assert(require.resolve('.', { paths: ['../'] }).endsWith('index.js'));
```

Comment:
Can we use `assert.match` to get better output in case of error – also, let's include slash/backslash in the check while we're at it
```suggestion
  assert.match(require.resolve('.', { paths: ['../'] }), /[/\\]index.js$/);
```

Reply:
definitely! :rocket: